<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlUI</class>
 <widget class="QWidget" name="ControlUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>391</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>水体液位仪设备控制</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../MyCPPProject/Win32/Image/png/Weather.ico</normaloff>../../MyCPPProject/Win32/Image/png/Weather.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(77,77, 77);color:white</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>45</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_time">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>设备时间</string>
     </property>
     <widget class="QDateTimeEdit" name="RdateTimeEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2018</year>
        <month>9</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QPushButton" name="RTimeBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>读取时钟</string>
      </property>
     </widget>
     <widget class="QPushButton" name="WTimeBtn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>设置时钟</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_DNO">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>设备号</string>
     </property>
     <widget class="QPushButton" name="RDNOBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>读取设备号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="WNOBtn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>设置设备号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_DNO">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_HOFFSET">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>210</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>水位偏移</string>
     </property>
     <widget class="QPushButton" name="WHOFFSETBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_HOFFSET">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>单位（米）</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_CAPINTERVAL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>远程升级</string>
     </property>
     <widget class="QPushButton" name="WCAPINTERVALBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>远程升级</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_RESET">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>采集器</string>
     </property>
     <widget class="QPushButton" name="RESETBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>重启</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_down">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>数据补抄</string>
     </property>
     <widget class="QDateTimeEdit" name="BdateTimeEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2018</year>
        <month>9</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="EdateTimeEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2018</year>
        <month>9</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QPushButton" name="DownBtn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>补抄数据</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_SNAPSHOT">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>手动采集</string>
     </property>
     <widget class="QPushButton" name="WSNAPSHOTBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>采集</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="TiltleLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../MyCPPProject/Win32/Image/png/control.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="TitleLabel_Name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>15</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>水体液位仪</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>46</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QScrollArea" name="ScrollArea_MinBtn">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>3</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ScrollAreaWidgetContents_MinBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>18</width>
        <height>18</height>
       </rect>
      </property>
      <widget class="QPushButton" name="MinBtn">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>1</y>
         <width>18</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QScrollArea" name="ScrollArea_CloseBtn">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>3</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ScrollAreaWidgetContents_CloseBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>18</width>
        <height>18</height>
       </rect>
      </property>
      <widget class="QPushButton" name="CloseBtn">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>1</y>
         <width>18</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_DNO</tabstop>
  <tabstop>RDNOBtn</tabstop>
  <tabstop>WNOBtn</tabstop>
  <tabstop>RdateTimeEdit</tabstop>
  <tabstop>RTimeBtn</tabstop>
  <tabstop>WTimeBtn</tabstop>
  <tabstop>WCAPINTERVALBtn</tabstop>
  <tabstop>RESETBtn</tabstop>
  <tabstop>WSNAPSHOTBtn</tabstop>
  <tabstop>BdateTimeEdit</tabstop>
  <tabstop>EdateTimeEdit</tabstop>
  <tabstop>DownBtn</tabstop>
  <tabstop>lineEdit_HOFFSET</tabstop>
  <tabstop>WHOFFSETBtn</tabstop>
  <tabstop>MinBtn</tabstop>
  <tabstop>CloseBtn</tabstop>
  <tabstop>ScrollArea_MinBtn</tabstop>
  <tabstop>ScrollArea_CloseBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
